### ISSUE TITLE
Provide pure floss build flavor

### ISSUE THREAD

**Issue by ghost** [tanallnight/Blip#20]
Could you provide a gradle build flavor that does not need crashlytics? Crashlytics is non-free software -- either removing this requirement or using things like the floss ACRA would help a lot.

**Comment by tanallnight** [comment]
Yeah sure, I cant do it right now as I am busy with my exams, ill work on it in a couple of weeks
